<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageStore</class>
 <widget class="QWidget" name="MessageStore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Store</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="mEditedItemName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mAddMessageButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Add message prototype</string>
       </property>
       <property name="text">
        <string>add component</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/property-editor-add-component.png</normaloff>
         <normalon>:/icons/property-editor-add-component.png</normalon>
         <disabledoff>:/icons/property-editor-add-component_disabled.png</disabledoff>
         <disabledon>:/icons/property-editor-add-component_disabled.png</disabledon>:/icons/property-editor-add-component.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mRemoveMessageButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remove selected message prototype</string>
       </property>
       <property name="text">
        <string>remove component</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/property-editor-remove-component.png</normaloff>
         <normalon>:/icons/property-editor-remove-component.png</normalon>
         <disabledoff>:/icons/property-editor-remove-component.png</disabledoff>
         <disabledon>:/icons/property-editor-remove-component.png</disabledon>:/icons/property-editor-remove-component.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="mMessageList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mEmitMessageButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Emit Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
